test/Test/Indigo/Contracts/HomebaseLite/Metadata/Views.hs

```diff
@@ -35,19 +33,19 @@ test_proposalInfo =
     let uri = #proposal_uri :! URI "ipfs://proposal"
         choices = ["Zero", "One", "Two", "Three"]
     withSender holder do
-      call contract (Call @"Propose") (uri, #choices :! choices)
+      transfer contract $ calling (ep @"Propose") (uri, #choices :! choices)
     storage <- getStorage contract
     let proposals = sProposalsRPC storage
     proposal <- getBigMapValue proposals uri
-    Showing <$> callOffChainView contract "proposalInfo" (ViewParam uri)
-      @@== Showing proposal
+    Showing <$> callOffChainView @ProposalInfo contract "proposalInfo" (ViewParam uri)
+      @@== Showing (toRPCProposalInfo proposal)
     res <- attempt $
       callOffChainView @ProposalInfo
         contract "proposalInfo" (ViewParam $ #proposal_uri :! URI "ipfs://nonexistent")
     case res of
-      Left (WithCallStack _ err) -> case fromException err of
-        Just (CustomTestError txt)
-          | "Reached FAILWITH instruction with \"NoSuchProposal\"" `isInfixOf` txt -> pass
-        _ -> failure $
-          "Expected a noSuchProposal error, but got " <> fromString (displayException err)
+      Left (WithCallStack _ err) ->
+        let errorStr = displayException err in
+        if "failed with: \"NoSuchProposal\"" `isInfixOf` (fromString errorStr)
+          then pass
+          else failure ("Expected a noSuchProposal error, but got: " <> (fromString errorStr))
```

Comment:
I wonder if we can use `expectTransferFailure` here now?

Reply:
Seems so. Have changed to use it.